Dockerfile.composer.no-dev

```diff
@@ -0,0 +1,12 @@
+FROM composer:1.6.3
+
+COPY composer.json \
+    composer.lock \
+    ./
+
+RUN composer --no-interaction install --no-dev --ignore-platform-reqs --no-autoloader --no-scripts --no-suggest
```

Comment:
`--ignore-platform-reqs` is what is key here, because it's what normally prevents using a generic image to run composer commands. I assume a missing extension will be caught when tests are run in the image.